src/modules/checkCompleted.js

```diff
@@ -0,0 +1,8 @@
+const knowCheckValue = (box) => {
+  if (box.checked) {
+    return true;
+  }
+  return false;
+};
+
```

Comment:
- Hey, great job with the status update, however, when I reload the page, all the checked box loses their states to become unchecked but are well updated in the localStorage. As this is not a good user experience, please, kindly make this little adjustment so that the checked boxes stay checked on reload.

Reply:
### Package.json file

- Hey, there's no script to start your server, when I tried starting your app, it throws the error in the terminal. You can add these lines to update your scripts below:


```js
"scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "start": "webpack serve --open"
  },
```